README.md

```diff
@@ -82,25 +100,41 @@ from [Frontegg Portal Domain](https://portal.frontegg.com/development/settings/d
 ```groovy
     dependencies {
     // Add the Frontegg Android Kotlin SDK
-    implementation 'com.frontegg.sdk:android:1.+'
+    implementation 'com.frontegg.sdk:android:LATEST_VERSION'
+    // Add Frontegg observables dependency
+    implementation 'io.reactivex.rxjava3:rxkotlin:3.0.1'
+}
+```
+```kotlin
+    dependencies {
+    // Add the Frontegg Android Kotlin SDK
+    implementation ("com.frontegg.sdk:android:LATEST_VERSION")
     // Add Frontegg observables dependency
     implementation 'io.reactivex.rxjava3:rxkotlin:3.0.1'
 }
 ```
 
 ### Set minimum sdk version
 
-To set up your Android minimum sdk version, open root gradle file at`android/build.gradle`,
+To set up your Android minimum sdk version, open the root gradle file at`android/build.gradle`,
 and add/edit the `minSdkVersion` under `buildscript.ext`:
```

Comment:
I think these instructions might be outdated. I think that `android {  }` is fine at least for Kotlin Gradle files these days. Not sure about Groovy.